nextflow.config

```diff
@@ -328,6 +329,10 @@ dag {
     enabled = true
     file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
 }
+prov {
+    enabled = true
```

Comment:
That's what I said to @ewels and he said, you'd just need to add 
prov.enabled = false in a config file to disable, so no need for a params.
And I kind of agreed because that's what we do with the other reports

Reply:
I'd say no, prov is a nextflow thingy, so `-prov false` maybe?